File: features/top/top/Page.go
package top

type Page struct {
	P	int32	`json:"p" name:"p" title:"分页位置"`
	N	int32	`json:"n" name:"n" title:"单页记录数"`
	Count	int32	`json:"count" name:"count" title:"总页数"`
	Total	int32	`json:"total" name:"total" title:"总记录数"`
	TopId	int64	`json:"topId" name:"topid" title:"顶部ID"`
}

File: features/wx/ts/Page.ts
import { int32 } from './lib/less';

/**
 * 分页
 */
export interface Page {
    /**
     * 分页位置
     */
    p: int32
    /**
    * 单页记录数
    */
    n: int32
    /**
     * 总页数
     */
    count: int32
    /**
     * 总记录数
     */
    total: int32
}
